tests/manage/mcg/test_mcg_resources_disruptions.py

```diff
@@ -116,6 +115,7 @@ def test_scale_down_rgw(self, scale_down_to):
         )
         self.cl_obj.wait_for_noobaa_health_ok()
 
+    @tier3
```

Comment:
I do not think this is the correct tier for this test - it checks that all MCG pods manage to migrate to another node when the one they rely on is drained. I do not see this as a negative test - it's more of tier 2 or 4

Reply:
The reason for moving it to tier3 is that this test case is not a disruptive one. Drain operation should not cause down time

Reply:
Resolving after re-reading tier3's definition